src/components/viewer/GalleryViewer/GalleryDocs/index.js

```diff
@@ -27,46 +30,51 @@ const defaultProps = {
 /*
  * Gallery viewer component for Docs [galleryDocsMimeType]
  */
-export default class GalleryDocs extends React.PureComponent {
-    static propTypes = propTypes;
-    static defaultProps = defaultProps;
-
-    render() {
-        const {
-            className: classNameFromProps,
-            docUrl,
-            mimeType,
-            canShowIframe,
-            notHttps,
-        } = this.props;
+function GalleryDocs(props) {
+    const {
+        className: classNameFromProps,
+        docUrl,
+        mimeType,
+        canShowIframe,
+        notHttps,
+    } = props;
 
-        if (!docUrl) {
-            return null;
-        }
+    if (!docUrl) {
+        return null;
+    }
 
-        const useGoogle = mimeType !== 'application/pdf' || !canShowIframe || notHttps;
-        const src = useGoogle
-            ? createUrlForGoogleViewer(docUrl)
-            : docUrl;
-        const sandbox = useGoogle
-            ? 'allow-scripts allow-same-origin allow-popups'
-            : undefined;
-
-        const className = _cs(
-            classNameFromProps,
-            'gallery-docs',
-            styles.galleryDocs,
-        );
+    const className = _cs(
+        classNameFromProps,
+        'gallery-docs',
+        styles.galleryDocs,
+    );
 
+    // NOTE: Google also support pdf for viewing but it does not work for localhost
+    const digestablePdf = mimeType === 'application/pdf' && canShowIframe && !notHttps;
+    if (digestablePdf) {
         return (
             <div className={className}>
-                <iframe
+                <embed
                     className={`doc ${styles.doc}`}
-                    title={docUrl}
-                    src={src}
-                    sandbox={sandbox}
+                    type="application/pdf"
+                    src={docUrl}
                 />
             </div>
         );
     }
+
+    return (
+        <div className={className}>
+            <iframe
+                className={`doc ${styles.doc}`}
```

Comment:
use _cs

Reply:
Didn't want to refactor old component :D